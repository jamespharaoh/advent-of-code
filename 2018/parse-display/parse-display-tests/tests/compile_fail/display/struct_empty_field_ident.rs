use parse_display::Display;

#[derive(Display)]
#[display("{}")]
struct TestStruct;

fn main() {}
