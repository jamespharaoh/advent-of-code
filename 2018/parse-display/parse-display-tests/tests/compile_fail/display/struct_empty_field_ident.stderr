error: {} is not allowed in struct format.
 --> $DIR/struct_empty_field_ident.rs:4:11
  |
4 | #[display("{}")]
  |           ^^^^
